.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,89 @@
+name: 🪳 Bug report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+# assignees:
+#   - OneSignal/ios-sdk
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Provide a thorough description of whats going on.
+      placeholder: e.g. The latest version of the SDK causes my screen to go blank when I tap on the screen three times.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce?
+      description: Provide as much detail as posible to reproduce the issue.
+      placeholder: |
+        1. Install vX.Y.Z of dependency
+        2. Launch the app on iOS device
+        3. Tap the screen three times
+        4. Note that the app crashes
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-are-expectations
+    attributes:
+      label: What did you expect to happen?
+      description: Also tell us, what did you expect to happen?
+      placeholder: I expected the app to continue running no matter how many times I tap the screen.
+    validations:
+      required: true
+  - type: input
+    id: ios-sdk-version
+    attributes:
+      label: OneSignal iOS SDK version
+      description: What version of the OneSignal iOS SDK are you using?
+      placeholder: Release 3.10.3
+    validations:
+      required: true
+  - type: dropdown
+    id: ios-major-version
+    attributes:
+      label: iOS version
+      description:  Which versions of iOS are broken for you?
+      multiple: true
+      options:
+        - "15"
+        - "14"
+        - "13"
+        - "12"
+        - "11 or below"
+    validations:
+      required: true
+  - type: textarea
+    id: ios-specific-version
+    attributes:
+      label: Specific iOS version
+      description: What are the specific versions.
+      placeholder: |
+        * iOS 15.4
+        * iOS 13.7
+        * iOS 8.0
+      render: Markdown
+    validations:
+        required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](#insert-project-code-of-conduct-here)
```

Comment:
I couldn't find a code of conduct for the repo.

Reply:
We could upload one to a repo in the OneSignalDevelopers org and use that as a standard link.

Reply:
Yes, that can work. Do you have a link to the code of conduct you'd like to use? I can create a `sdks` repo and dump shared resources there.

cc/ @emawby @adamschlesinger @jkasten2

Reply:
Some variant of this would be appropriate: https://coda.io/d/Engineering_dFl06o0dE6E/Code-of-Conduct_susys

Happy to review and/or help draft.

Reply:
I'm down with it. I think it's safe to land this PR and follow up with PR to update the code of conduct. That way we don't hold this up.

Reply:
@iAmWillShepherd I am down for that. Can you remove the "Code of Conduct" reference then?, as it doesn't link to anything they can agree to.

Reply:
I could, but then the team leads for each repo these forms are deployed at will also need to go and update their issue forms. I know of 3/4 repos (so 9/12 forms in total assuming Android is deployed) that will need to be updated if we make this change. At the moment, clicking the link is essentially a no-op.

Reply:
I'm going to use the contributing.md doc as suggested by @jkasten2 in [Slack](https://onesignal.slack.com/archives/C3APVHPCG/p1653073586626929?thread_ts=1653072627.157999&cid=C3APVHPCG)

Reply:
@jkasten2 I addressed your feedback in 
* https://github.com/OneSignal/OneSignal-iOS-SDK/pull/1080/commits/db0a65ba2648729b322e7792a29863e72918e92a
* https://github.com/OneSignal/OneSignal-iOS-SDK/pull/1080/commits/958438bf87138a060b4e58a861c58e5de89b3ead
* https://github.com/OneSignal/OneSignal-iOS-SDK/pull/1080/commits/d45415117ab948f5fd8fb94fcc0d3ee06f66b158